src/main/resources/fabric.mod.json

```diff
@@ -28,11 +28,5 @@
     "java": ">=17"
   },
   "suggests": {
```

Comment:
The "suggests" context is unneeded as it has no data for it.

Reply:
Honestly, I just didn't want to change the `fabric.mod.json` file too much. I could probably add `"breaks"` as well while I'm at it.

Reply:
fixed with the latest commit